src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py

```diff
@@ -422,7 +424,9 @@ def test_monitor_log_analytics_workspace_data_export(self, resource_group, accou
                      '--destination {sa_id_1} --enable -t "SecurityEvent Heartbeat"',
                      checks=[
                      ])
-        with self.assertRaisesRegex(HttpResponseError, 'you can create 10 export rules to 10 different destinations'):
+        #with self.assertRaisesRegex(HttpResponseError, 'you can create 10 export rules to 10 different destinations'):
+        with self.assertRaisesRegex(HttpResponseError,
+                                        'You are adding a destination that is already defined in rule: clitest. Destination must be unique across export rules in your workspace . See http://aka.ms/LADataExport#limitations'):
```

Comment:
```suggestion
                                    'You are adding a destination that is already defined in rule: clitest. Destination must be unique across export rules in your workspace . See http://aka.ms/LADataExport#limitations'):
```
Please pay attention to code alignment and delete the unused code.